pkg/db/memory_test.go

```diff
@@ -357,7 +407,12 @@ func TestMemoryDb_ReverseLookup(t *testing.T) {
 
 func TestMemoryDb_OrganizationById(t *testing.T) {
 	t.Run("organization is found", func(t *testing.T) {
-		eventSystem, db := initDb()
+		repo, err := test.NewTestRepo(t.Name())
```

Comment:
Can some of this logic be put outside of the `t.Run` calls?

Reply:
Wrapped it in an initializer function